test/streams/streams_controller_test.rb

```diff
@@ -37,4 +37,12 @@ class Turbo::StreamsControllerTest < ActionDispatch::IntegrationTest
 STREAM
     end
   end
+  
+  test "render correct partial for namespaced models" do
+    get users_profile_path(id: 1), as: :turbo_stream
+    assert_equal <<-STREAM, @response.body
```

Comment:
Since this test inherits from ActionDispatch::IntegrationTest, we should have access to [assert_dom_equals](https://github.com/rails/rails-dom-testing#dom-assertions).

If that's the case, let's use it so that the test isn't dependent on the attribute order:

```ruby
test "render correct partial for namespaced models" do
  get users_profile_path(id: 1), as: :turbo_stream
  assert_dom_equal <<~HTML, @response.body
    <turbo-stream action="replace" target="users_profile_1"><template><p>User</p></template></turbo-stream>
    <turbo-stream action="update" target="users_profile_1"><template><p>User</p></template></turbo-stream>
  HTML
end
```

Reply:
I can change that, but I mimicked the other tests. Do these need to be changed to use `assert_dom_equal` as well?

Reply:
I can touch up the others in a separate commit.